docs/service-catalog/docs/014-details-etcd-database.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Etcd Database
+type: Details
+---
+
+## Overview
+
+The Service Catalog requires an `etcd` database cluster for a production use. 
+It has a separate `etcd` cluster defined in the Service Catalog [etcd][sc-etcd-sub-chart] sub-chart. 
+The [etcd-operator][etcd-operator] installs and manages the `etcd` clusters deployed to Kubernetes,
+and also automates tasks related to operating an `etcd` cluster, for example executing [backup][etcd-backups] and [restore][etcd-restores] procedures. 
+
+> **NOTE:** The [etcd-operator][etcd-operator] is Namespace-scoped and is installed only in `kyma-system` Namespace.
+
+## Details
+
+This section describes the backup and restore processes of the `etcd` cluster for the Service Catalog.
+
+### Backup
+
+To execute the backup process the following values need to be set in the [core][core-chart-values] chart:
```

Comment:
`To execute the backup process, the following values must be set in the [core][core-chart-values] chart:`